samza-sql/src/main/java/org/apache/samza/sql/data/SamzaSqlRelMessage.java

```diff
@@ -113,13 +119,13 @@ public static SamzaSqlRelMessage createRelMessage(List<Object> fieldValues, List
     int keyIndex = fieldNames.indexOf(KEY_NAME);
```

Comment:
nitpick: order of parameters: In all the other methods you have fieldNames first, why here the order is different?